src/Illuminate/Cache/Repository.php

```diff
@@ -35,7 +35,7 @@ class Repository implements CacheContract, ArrayAccess
     /**
      * The default number of minutes to store items.
      *
-     * @var int
+     * @var float
```

Comment:
No, a `float` is fundamentally different from `int`.

* `number` is the same as writing `float|int`.
* `integer` and `long` are both aliases for `int`
* there is no such thing as `short` in php
* `double` and `real` are both aliases for `float`